### ISSUE TITLE
Have Symbiote support operations other than flash 

### ISSUE THREAD

**Issue by moredip** [moredip/Frank#75]
AS A test automater
SO THAT I can experiment with how different selectors might work
I WOULD LIKE to use Symbiote to perform custom operations (not just flash)

The main example would be to call 'touch' rather than 'flash', but there are probably a few others that'd be helpful for testing

**Comment by ondrejhanslik** [comment]
Frank console already handles this use case.

**Comment by moredip** [comment]
That's true, but some users find Symbiote more convenient than the console. It'd be nice to allow those users to stick in the tool they prefer.

Cheers,

## Pete

Typed on a little bitty keyboard
